src/2_RTB/VerbaleInterno_071123/content.typ

```diff
@@ -20,39 +20,39 @@ Su proposta dei componenti è stata assegnata questa prima distribuzione dei ruo
 - Amministratore: Matteo Rango;
 - Analista: Simone Caregnato;
 - Analista: Nancy Kalaj;
-- Verificatore: Riccardo Constantin;
+- Verificatore: Riccardo Costantin;
 - Verificatore: Giacomo D'Ovidio;
-Per il momento i ruoli di Progettista e Programmatore non sono stati assegnati, in quanto non sono significativi per questa fase di progetto.
+Per il momento i ruoli di Progettista e Programmatore non sono stati assegnati, in quanto per il momento hanno operatività limitata.
 
 == Verbali degli Stand-Up
-Dato l'importanza di registrare le decisioni prese durante gli incontri, si è sollevato il dibattito sulla necessità di verbalizzare gli Stand-Up. La decisione è stata quella di verbalizzare solo gli incontri in cui sono presenti quattro o più partecipanti, che quindi coinvolgano la maggior parte del gruppo di lavoro; questo perché si tratterà di incontri più importanti, in cui si prenderanno decisioni più rilevanti. Questa prassi dovrà essere aggiunta alle #glossary("NdP") e il compito è stato assegnato a Matteo Rango.
+Data l'importanza di registrare le decisioni prese durante gli incontri, si è sollevato il dibattito sulla necessità di verbalizzare gli Stand-Up. La decisione è stata quella di verbalizzare solo gli incontri Stand-Up in cui siano state prese le decisioni più importanti, di conseguenza con la presenza di quattro o più partecipanti, che quindi coinvolgano la maggior parte del gruppo di lavoro. Questa prassi dovrà essere aggiunta alle #glossary("NdP") e il compito è stato assegnato a Matteo Rango.
```

Comment:
La decisione è stata quella di verbalizzare solo gli incontri Stand-Up in cui siano state prese le decisioni più importanti, di conseguenza con la presenza di quattro o più partecipanti, che quindi coinvolgano la maggior parte del gruppo di lavoro. 

La decisione è stata quella di verbalizzare solo gli incontri Stand-Up in cui vengono prese decisioni rilevanti. 
In questo contesto il team ha stabilito come necessaria la presenza di almeno 4 membri al fine di prendere una qualsiasi  decisione, ne risulta che solamente gli Stand-Up che coinvolgono almeno 4 componenti possono essere documentanti. 

[A me piace di più così ma valuta tu, senza fastidi]

Reply:
La decisione è stata quella di verbalizzare esclusivamente gli incontri Stand-Up in cui si affrontano decisioni di particolare rilevanza. Si è concordato che, affinché una  decisione sia presa, è  necessaria la presenza di almeno 4 membri del team. Pertanto, saranno documentati soltanto gli Stand-Up ce coinvolgono almeno 4 componenti, assicurando così la registrazione delle decisioni più significative con la partecipazione della maggior parte del gruppo.

Cosa ne pensi?

Reply:
> La decisione è stata quella di verbalizzare solo gli incontri Stand-Up in cui siano state prese le decisioni più importanti, di conseguenza con la presenza di quattro o più partecipanti, che quindi coinvolgano la maggior parte del gruppo di lavoro.
> 
> La decisione è stata quella di verbalizzare solo gli incontri Stand-Up in cui vengono prese decisioni rilevanti. In questo contesto il team ha stabilito come necessaria la presenza di almeno 4 membri al fine di prendere una qualsiasi decisione, ne risulta che solamente gli Stand-Up che coinvolgono almeno 4 componenti possono essere documentanti.
> 
> [A me piace di più così ma valuta tu, senza fastidi]

La decisione è stata quella di verbalizzare esclusivamente gli incontri Stand-Up in cui si affrontano decisioni di particolare rilevanza. Si è concordato che, affinché una  decisione sia presa, è  necessaria la presenza di almeno 4 membri del team. Pertanto, saranno documentati soltanto gli Stand-Up ce coinvolgono almeno 4 componenti, assicurando così la registrazione delle decisioni più significative con la partecipazione della maggior parte del gruppo.

Cosa ne pensi?

Reply:
Pertanto, saranno documentati soltanto gli Stand-Up che coinvolgono almeno 4 componenti, assicurando così la registrazione delle decisioni più significative.

Lascerei così